Formula/sonarqube-lts.rb

```diff
@@ -1,8 +1,8 @@
 class SonarqubeLts < Formula
   desc "Manage code quality"
   homepage "https://www.sonarqube.org/"
-  url "https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-7.9.6.zip"
-  sha256 "9991d4df42c10c181005df6a4aff6b342baf9be2f3ad0e83e52a502f44d2e2d8"
+  url "https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-8.9.6.50800.zip"
+  sha256 "b2911b7420e656fbb94880af76ff9cd26ec2f13d26923a6cb4174a2ea9457d3d"
```

Comment:
```suggestion
  url "https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-8.9.7.52159.zip"
  sha256 "ce528344a384d0ee5b6ad44b242005a0053914683311ea967c43ff86c81fcc94"
```

The newest LTS version is now 8.9.7.52159.

Reply:
Do we have a preferred way of integrating changes to BrewTestBot PRs (e.g., create and commit suggestions, amend the commit locally and push to the branch, or something else)?

Reply:
Whatever you want